app/scripts/controllers/editProduct.js

```diff
@@ -39,7 +51,7 @@ angular.module('endslaverynowApp').controller('EditProductCtrl', [
         var product = $scope.dataRepository.getProductById($scope.productId);
         $scope.name = product.getName();
         $scope.description = product.getDescription();
-        $scope.ranking = product.getRanking();
+        // $scope.ranking = product.getRanking();
```

Comment:
Any reason this was commented out?

Since we're no longer affiliated with ESN, we could possibly rethink what the rating section looks like. Though this is a separate task

Reply:
I honestly can't remember. I'll put it back in for now. I wonder if I meant to comment it out temporarily and forgot?

Reply:
Ah, I've investigated and found that products don't have a ranking! I can't remember the history, but the product model has no ranking property, and the getRanking() method is hard-coded to return null.
I'll still uncomment it, though.